.ci/helper_shellchecks.sh

```diff
@@ -10,8 +10,13 @@ mapfile -t shellscript_locations < <({ git grep -lE '^#!(/usr)?/bin/(env )?(bash
 SHELLSCRIPT_ERROR=0
 
 for shellscript in "${shellscript_locations[@]}"; do
-    echo -e "${ANSI_GREEN}Running shellcheck on ${shellscript}"
-    shellcheck "${shellscript}" || SHELLSCRIPT_ERROR=1
+    # These two scripts are running on the device, so they should use pure sh
```

Comment:
It gives me "No issues detected!" I don't really know if it's POSIX-compliant or not but it works on Linux, BSD, and BusyBox. In any case, I see that `ps -p` gives out appropriate error codes without any grep involvement, which all seems more elegant to me. ;-)